### PR TITLE
Cleanup legacy fact usage

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: manifests/params.pp [hunk 1]
```diff
@@ -45,14 +45,8 @@
...
     'LinuxMint': {
-      case fact('lsbmajdistrelease') {
-        # Linux Mint 18* is based on Ubuntu 16.04
-        default: {
-          $origins            = [
-            '${distro_id}:${distro_codename}-security', #lint:ignore:single_quote_string_with_variables
-          ]
-        }
-      }
+      # Linux Mint 18* is based on Ubuntu 16.04
```
Comment (ekohl) [inline manifests/params.pp:48]: Is this comment still relevant?

File: manifests/params.pp [hunk 2]
```diff
@@ -17,9 +17,9 @@
     'force_confmiss'       => false,
   }
 
-  case fact('lsbdistid') {
+  case downcase($facts['os']['name']) {
```
Comment (ekohl) [inline manifests/params.pp:20]: Is this really the same thing? I'm not too familiar with the LSB facts and always avoided them if I could.

Comment (bastelfreak) [reply manifests/params.pp:20]: yes it is:
```
root@basteles-bastelknecht ~ # puppet facts show os
{ 
  "os": {
    "architecture": "amd64",
    "distro": {
      "codename": "bullseye",
      "description": "Debian GNU/Linux 11 (bullseye)",
      "id": "Debian",
      "release": {
        "full": "11.3",
        "major": "11",
        "minor": "3"
      }
    },
    "family": "Debian",
    "hardware": "x86_64",
    "name": "Debian",
    "release": {
      "full": "11.3",
      "major": "11",
      "minor": "3"
    },
    "selinux": {
      "enabled": false
    }
  }
}
root@basteles-bastelknecht ~ # puppet facts show lsbdistid
{
  "lsbdistid": "Debian"
}
root@basteles-bastelknecht ~ #
```

File: manifests/params.pp (hunk 1 above)
Comment (bastelfreak) [reply manifests/params.pp]: mhm probably not, since we don't have anything mint 18 specific. I deleted it.

**Review by ekohl — APPROVED** [review]

**Pull request merged**